src/components/landing/Projects.vue

```diff
@@ -450,29 +480,39 @@ export default {
       this.activeSlide = num;
     },
     setPrev: function () {
+      this.prevSlide = this.activeSlide;
       if (this.activeSlide === 1) {
         this.activeSlide = 10;
       } else {
         this.activeSlide = this.activeSlide - 1;
       }
     },
     setNext: function () {
+      this.prevSlide = this.activeSlide;
       if (this.activeSlide === 10) {
         this.activeSlide = 1;
       } else {
         this.activeSlide = this.activeSlide + 1;
       }
     },
   },
+  computed: {
+    direction: function () {
+      if (this.prevSlide > this.activeSlide) {
+        return "right";
+      } else return "left";
+    },
+  },
   watch: {
     activeSlide: {
       immediate: true,
       handler(newSlide, oldSlide) {
         clearInterval(interval);
         interval = setInterval(() => {
-          this.setActive(newSlide + 1);
-          if (this.activeSlide > 10) {
+          if (newSlide + 1 > 10) {
             this.setActive(1);
+          } else {
+            this.setActive(newSlide + 1);
```

Comment:
```suggestion
          this.setActive(newSlide + 1 > 10 ? 1 : newSlide + 1);
```

Can be simplified like this. Also, where is this `10` coming from?

Reply:
Indeed. `10` is the hardcoded length of the slides.